docs/installation/server-config/index.rst

```diff
@@ -281,7 +288,7 @@ When deploying KSQL to production, the following settings are recommended in you
 
     # Set the batch expiry to Long.MAX_VALUE to ensure that queries will not
     # terminate if the underlying Kafka cluster is unavailable for a period of
-    # time.  
+    # time.
     producer.confluent.batch.expiry.ms=9223372036854775807
```

Comment:
We should prefix these

Reply:
@rodesai as in `kafka.streams.`?

Reply:
Yep. That's a weird looking setting though. Why does it have confluent in the name?